galley/pkg/config/analysis/analyzers/auth/mtls/policy_checker.go

```diff
@@ -56,69 +49,115 @@ func (w TargetService) FQDN() string {
 	return w.fQDN
 }
 
-// PortNumber is the port used by the service. If set (non-zero), then
-// PortName must be the default value ("").
+// PortNumber is the port used by the service.
 func (w TargetService) PortNumber() uint32 {
 	return w.portNumber
 }
 
-// PortName is the name of the port used by the service. If set (not ""),
-// then PortNumber must be the default value (0).
-func (w TargetService) PortName() string {
-	return w.portName
+func (w TargetService) String() string {
+	if w.PortNumber() != 0 {
+		return fmt.Sprintf("%s:%d", w.fQDN, w.portNumber)
+	}
+	return w.FQDN()
```

Comment:
Should be consistent one way or another in these statements between using `w.fQDN` and `w.FQDN()`